Assets/Plugins/Colyseus/Client.cs

```diff
@@ -169,7 +169,7 @@ void OnJoin()
 			room.OnError += OnError;
 			room.OnJoin += OnJoin;
 
-			ColyseusManager.Instance.AddRoom(room);
+			_ = ColyseusManager.Instance.AddRoom(room);
```

Comment:
IMO: not really good to just declare an unused variable.

Reply:
The underscore indicates that we're aware this is an async method, and that its result should be discarded (https://docs.microsoft.com/en-us/dotnet/csharp/discards). It looks like the desired behavior here is to fire and forget the async method, so this cleans up the warning related to that.